### ISSUE TITLE
factorio experimental branch broken

### ISSUE THREAD

**Issue by Frisasky** [GameServerManagers/LinuxGSM#2452]
## User Story

It seems factorio's experiment branch support are broken again

The latest version of experiment branch is 0.17.58 

## Further Information
```
fctrserver@hivemind:~$ ./fctrserver update
[ .... ] Update fctrserver: Checking for update: factorio.com: checking local bu[  OK  ] Update fctrserver: Checking for update: factorio.com: checking local bu[ .... ] Update fctrserver: Checking for update: factorio.com: checking remote b[  OK  ] Update fctrserver: Checking for update: factorio.com: checking remote b[  OK  ] Update fctrserver: Checking for update: factorio.com

Update available
* Local build: 0.16.51 linux64
* Remote build: 1.7.2 linux64

applying update...
    fetching command_stop.sh...OK
[ INFO ] Stopping fctrserver: Check IP: 103.212.224.46
[  OK  ] Stopping fctrserver: Graceful: CTRL+c: 2: OK

curl: (22) The requested URL returned error: 404 NOT FOUND
downloading factorio_headless_linux64-1.7.2.tar.xz...OK
extracting factorio_headless_linux64-1.7.2.tar.xz...OK
copying to /home/fctrserver/serverfiles...cp: cannot stat '/home/fctrserver/lgsm/tmp/factorio/*': No such file or directory
FAIL
[  OK  ] Starting fctrserver: Factorio Server
    fetching alert.sh...OK
    fetching info_messages.sh...OK
fctrserver@hivemind:~$
```
## Expected behaviour

You know

**Comment by h3o66** [comment]
Afaik this was fixed with #2890 
Is this still the case ?

**Comment by h3o66** [comment]
Afaik this is fixed
Please reopen if this still persists
~ closing
